test/setup/build.ts

```diff
@@ -0,0 +1,35 @@
+import { execSync } from 'child_process';
+import { renameSync, rmSync } from 'fs';
+import { projectRoot } from '../utils';
+
+const rustGithubRepository = 'metaplex-program-library';
+const repostitoryDir = `${rustGithubRepository}-master`;
+
+const rustProgramsRepository = `https://github.com/metaplex-foundation/${rustGithubRepository}/archive/refs/heads/master.zip`;
+
+function build() {
+  const programs: string[] = [
+    'auction/program',
+    'token-metadata/program',
+    'token-vault/program',
+    'metaplex/program',
+  ];
+
+  process.chdir('test');
```

Comment:
It is usually advised to not change dir during a script or app run.
Instead just resolve all paths provided to tools and give them full path so it doesn't matter from which directory you run.